virttest/qemu_devices/qcontainer.py

```diff
@@ -2030,10 +2031,17 @@ def serials_define_by_variables(self, serial_id, serial_type, chardev_id,
         :param bus: the busid of parent bus(optional, virtio only)
         :param nr: the nr(port) of the parent bus(optional, virtio only)
         :param reg: reg option of isa-serial(optional)
+        :param bus_extra_params: extra virtio_serial_pci params
         :return: the device list that construct the serial device
         """
 
         devices = []
+        bus_params = {}
+        if bus_extra_params:
+            bus_extra_params_ = (_.split('=', 1) for _ in
```

Comment:
above code equal to below, let's update it to make it more readable.

bus_params = dict([_.split('=') for _ in bus_extra_params.split(',') if _])

Reply:
thanks and update.